test/e2e/flask/btc/btc-account-overview.spec.ts

```diff
@@ -63,7 +63,6 @@ describe('BTC Account - Overview', function (this: Suite) {
         assert.equal(await homePage.check_isBridgeButtonEnabled(), false);
         assert.equal(await homePage.check_isSwapButtonEnabled(), false);
         assert.equal(await homePage.check_isBuySellButtonEnabled(), true);
-        assert.equal(await homePage.check_isReceiveButtonEnabled(), true);
```

Comment:
Removing Receive button test. I believe this was the only instance

<img width="352" alt="Screenshot 2025-05-16 at 9 27 00 AM" src="https://github.com/user-attachments/assets/2d05069f-76d4-4e62-9560-e7a296efb121" />